### DISCUSSION TITLE
Wifi  not working

### DISCUSSION THREAD

**Discussion by ghost** [dl9rdz/rdz_ttgo_sonde#110] [TTGO firmware]
Hello, after updating  the newest development rep. on a  t-beam T22_V1.1   Wifi  will not work as it works  before.
Also with  NEW installation  no Connect to the  local network is possible.

AP Connect     worK#s  well
Wifi Scan show#s  my  network; all data  a correct  in the  config

OLDER  devel20xx   works !

NO CONNECT  with newest  development  devel20210630-B11-full.bin  

ANY  IDEA ??

**Comment by ghost** [comment]
SORRY........Now  i read  the Post  about this  knowing issue    and  changed  to WiFi  Mode  1     which works

HOPE  the Problem will resolved  in the  future

**Comment by dl9rdz** [comment]
OK, I will have a look at this problem later this week. Nothing changed, so it is a bit strange.  The only difference in Mode 1 is that it tries connecting forever until it succeeds, whereas mode 3 tries once and if not successfull switched to access point mode. Maybe the timeout while waiting to connect is a bit short. Although it works with my mobile AP and with my network at home.
